LayoutTests/imported/w3c/web-platform-tests/css/css-variables/variable-presentation-attribute-expected.txt

```diff
@@ -4,7 +4,7 @@ PASS Testing 'stroke-width' on '#box2'.
 PASS Testing 'stroke-width' on '#box3'.
 PASS Testing 'clip' on '#test4'.
 PASS Testing 'alignment-baseline'.
-PASS Testing 'baseline-shift'.
+FAIL Testing 'baseline-shift'. assert_equals: Default value. expected "baseline" but got "0px"
```

Comment:
Why this new failure?

Reply:
@smfr - this test is wrong on WPT upstream, it fails across the board. It should be expecting '0px' as per spec but it expect 'baseline' (old value).

Reply:
WPT test case - https://wpt.fyi/results/css/css-variables/variable-presentation-attribute.html?label=experimental&label=master&aligned